src/routing/comm.rs

```diff
@@ -30,18 +31,23 @@ pub(crate) struct Comm {
 }
 
 impl Comm {
-    pub fn new(
+    pub async fn new(
         transport_config: qp2p::Config,
         event_tx: mpsc::Sender<ConnectionEvent>,
     ) -> Result<Self> {
         let quic_p2p = QuicP2p::with_config(Some(transport_config), Default::default(), true)?;
 
         // Don't bootstrap, just create an endpoint where to listen to
         // the incoming messages from other nodes.
-        let endpoint = quic_p2p.new_endpoint()?;
+        let (endpoint, _, incoming_messages, disconnections) = quic_p2p.new_endpoint().await?;
```

Comment:
better refactor the above comment, also explain what's the ignored returned item and what disconnections for?